### PR TITLE
add tips for finding reviewers using @noamross' forum post

### PR DESCRIPTION
**Pull request by maelle**
cf #109

### CODE REVIEW

File: softwarereview_editor.Rmd [hunk 1]
```diff
@@ -63,6 +64,36 @@ Due date: YYYY-MM-DD
...
+
+Here are criteria to keep in mind when choosing a reviewer. You might need to piece this information together by searching CRAN and the potential reviewer’s GitHub page and general online presence (personal website, Twitter).
+
+* Has not reviewed a package for us within the last 6 months.
+* Some package development experience.
+* Some domain experience.
+* No [conflicts of interest](#coi).
+* Try to balance your sense of the potential reviewer’s experience against the complexity of the package.
+* Diversity - with two reviewers both shouldn’t be cis white males.
+* Some evidence that they are interested in openness or R community activities although blind emailing is fine.
```
Comment (noamross) [inline softwarereview_editor.Rmd]:
```suggestion
* Some evidence that they are interested in openness or R community activities, although blind emailing is fine.
```

File: softwarereview_editor.Rmd [hunk 2]
```diff
@@ -63,6 +64,36 @@ Due date: YYYY-MM-DD
...
+* You can also search for authors of related packages on [r-pkg.org](https://r-pkg.org/).
+* R-Ladies has a [directory](http://rladies.org/directory/) specifying skills and interests of people listed.
+
+#### Criteria for choosing a reviewer
+
+Here are criteria to keep in mind when choosing a reviewer. You might need to piece this information together by searching CRAN and the potential reviewer’s GitHub page and general online presence (personal website, Twitter).
+
+* Has not reviewed a package for us within the last 6 months.
+* Some package development experience.
+* Some domain experience.
```
Comment (noamross) [inline softwarereview_editor.Rmd]:
```suggestion
* Some domain experience in the field of the package or data source
```

File: softwarereview_editor.Rmd [hunk 3]
```diff
@@ -63,6 +64,36 @@ Due date: YYYY-MM-DD
...
+
+As a (guest) editor, use 
+
+* the potential suggestions made by the submitter(s), 
+* the Airtable database of reviewers and volunteers 
+* and the authors of [rOpenSci packages](https://ropensci.org/packages/). 
+
+When these sources of information are not enough, 
+* ping other editors in Slack for ideas, 
+* look for users of the package (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter).
```
Comment (noamross) [inline softwarereview_editor.Rmd]:
```suggestion
* look for users of the package or of the data source/upstream service the package connects to (via their opening issues in the repository, starring it, citing it in papers, talking about it on Twitter). 
```

File: softwarereview_editor.Rmd [hunk 4]
```diff
@@ -63,6 +64,36 @@ Due date: YYYY-MM-DD
 -   Switch numbered tag `to 3/reviewers-assigned` once reviewers are assigned.
 -   Invite authors and reviewers to rOpenSci Slack if they aren't on this Slack already.
 
+#### How to look for reviewers
+
+##### Where to look for reviewers?
+
+As a (guest) editor, use 
+
+* the potential suggestions made by the submitter(s),
```
Comment (noamross) [inline softwarereview_editor.Rmd]:
```suggestion
* the potential suggestions made by the submitter(s), (although submitters may have a narrow view of the types of expertise needed.  We suggest not using more than one of suggested reviewers)
```

File: softwarereview_editor.Rmd [hunk 5]
```diff
@@ -63,6 +64,36 @@ Due date: YYYY-MM-DD
...
+
+* Has not reviewed a package for us within the last 6 months.
+* Some package development experience.
+* Some domain experience.
+* No [conflicts of interest](#coi).
+* Try to balance your sense of the potential reviewer’s experience against the complexity of the package.
+* Diversity - with two reviewers both shouldn’t be cis white males.
+* Some evidence that they are interested in openness or R community activities although blind emailing is fine.
+
+Each submission should be reviewed by _two_ package reviewers. Although it is fine for one of them to have very little package development experience and more domain knowledge, the review should not be split in two, both reviewers need to review the package comprehensively, though from their particular perspective.
```
Comment (noamross) [inline softwarereview_editor.Rmd]:
```suggestion
Each submission should be reviewed by _two_ package reviewers. Although it is fine for one of them to have less package development experience and more domain knowledge, the review should not be split in two.  Both reviewers need to review the package comprehensively, though from their particular perspective.  In general, at least one reviewer should have prior reviewing experience, and of course inviting one new reviewer expands our pool of reviewers.
```

**Comment by maelle** [comment]
>  These in-line PR edits make editing text in git so much less of pain.

Yesss they do.

Thanks for your edits! I'll wait to see if other editors want to chime in.

**Pull request merged**